client/src/components/categorical/value/index.js

```diff
@@ -297,7 +313,7 @@ class CategoryValue extends React.Component {
     height
   ) => {
     /*
-      Knowing that colorScale is based off continuous data, 
+      Knowing that colorScale is based off continuous data,
```

Comment:
Was this a visual catch or was it an extension? I'd like to be able to catch these too :)

Reply:
Oh great question! VSCode has an option to trim that 😄 

`"files.trimTrailingWhitespace": true,`

https://github.com/prettier/prettier/issues/2422#issuecomment-313843775